test/index.js

```diff
@@ -160,7 +160,7 @@ describe('rollup-plugin-hash', () => {
 		const res = hashWithOptions({ dest: 'tmp/[hash].js', manifest: 'tmp/manifest.json' });
 		return res.then(() => {
 			const tmp = fs.readdirSync('tmp');
-			const manifest = require('./tmp/manifest.json');
```

Comment:
Not sure what you mean here.

Reply:
I don't see what improvements or changes this diff does. `require` already parses the JSON.

Reply:
@GerkinDev I've addressed the comments on this PR.

Reply:
@GerkinDev, looks like this wasn't a useless chore. Without these, the tests fail on Node 4 since require likely does not parse it in that old node version.

Reply:
Okay my bad. But since you changed node env (see [this diff](https://github.com/phamann/rollup-plugin-hash/pull/23/commits/e29f6ea2fb360387cf3c323eef175f26ab262500)), is it still relevant ?

Reply:
Good call, removed those changes. Should be good to merge now!